app/constants/owner.js

```diff
@@ -1,12 +1,11 @@
 const constants = {
   routes: {
-    name: {
-      get: '/register/owner/name',
-      post: '/register/owner/name'
+    home: {
+      get: '/'
     },
-    postcode: {
-      get: '/register/owner/postcode',
-      post: '/register/owner/postcode'
+    ownerDetails: {
+      get: '/register/owner/owner-details',
```

Comment:
Sorry just realised these should match the routes on the Mural board.

This one should be `cdo/create`.

Reply:
Done